doc/graph/rst/graph_supported_operations.rst

```diff
@@ -31,9 +31,11 @@ Supported Operations
    dev_guide_op_elubackward
    dev_guide_op_end
    dev_guide_op_exp
-   dev_guide_op_groupnorm
    dev_guide_op_gelu
    dev_guide_op_gelubackward
+   dev_guide_op_genindex
+   dev_guide_op_greaterequal
```

Comment:
This is a fix. Let's create a separate PR for it and get it fixed as soon as possible.

Reply:
OK, create a separate PR https://github.com/uxlfoundation/oneDNN/pull/3137